### ISSUE TITLE
every version released after april 1 has broken usnewson.com - most recent is the worst

### ISSUE THREAD

**Issue by infinitewaveparticle** [bromite/bromite#1093]
### Bromite version

Version: 90.0.4430.74
Arch: arm64
Android version: 9.0
Device model: NVIDIA Shield PPR1.180610.011

### Flags

These are the flags changed under `chrome://flags`:

### Is this bug about the SystemWebView?

No

### Is the bug reproducible with latest version?

Yes

### Can the bug be reproduced with corresponding Chromium version?

No

### Is this bug happening in an incognito tab?
both regular and igcognito

### Allow ads for the site; is the bug caused by the adblocker?
No

### Is the bug a crash?
yes, it is now

Yes, this is the crash dump: i dont have anything for adb and wont be getting it. 
```
<paste here logcat dump>
```

### Describe the bug
the first versions after the apr 1 version (i think 89.9.4368 or close to those numbers) just muted the live news video feeds on usnewson.com. the most recent version causes the screen to blink like a strobe light and crashes the app.

### Steps to reproduce the bug

Steps to reproduce the bug:
1. Go to usnewson.com
2. Click on any live news stream (fox, cnn, etc)
3. witness bug (which now requires force closing the app)

Do not write "any website", specify which URLs can be used to reproduce the issue.

### Expected behavior
like in the apr 1 version and MOST before (this has happened before) the videos should play as intended and have sound (sometimes you have to unmute the stream because it begins muted when it works in the apr 1 version)

### Screenshots

i am using Android TV (android 9 on nvidia shield), so no screenshots

**Comment by kokukyocho** [comment]
Hello @infinitewaveparticle!

I am a bot :robot:

Thanks for submitting this issue!
I noticed that it is missing the template, please edit the issue to match the template for either a [bug](https://github.com/bromite/bromite/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) or a [feature request](https://github.com/bromite/bromite/blob/master/.github/ISSUE_TEMPLATE/feature_request.md).

The issue will be automatically re-opened after that.

**Tip**: if you use GitHub in Desktop mode the template will be proposed to you when submitting issues.

**Comment by infinitewaveparticle** [comment]
@csagan Are you not interested in fixing this? Also, I checked how far back this website stopped functioning and my original posting was incorrect. I had to install version 89.0.4389.78 for the website to function again (which it only does in desktop mode for some reason). I think that this likely isn't the only website in the world to lose its functioning due to this obvious bug. No website should cause a crash and all video streams should function as intended... I believe that is a popular opinion.

Sent from ProtonMail mobile

-------- Original Message --------
